lib/model.js

```diff
@@ -106,7 +156,7 @@ var EventEmitter = require('eventemitter3'),
          * @return {Boolean}
          */
         isNew: function () {
-            return !this.id;
+            return !this.getId();
```

Comment:
yes, you are right. But i think better `this.getId() === null`. Id can change in runtime